### PR TITLE
Iiif docs

### PR DESCRIPTION
**Pull request by mjordan**
**GitHub Issue**: Maybe  https://github.com/Islandora/documentation/issues/1184 ?

# What does this Pull Request do?

Adds a page to the User Documentation describing how IIIF is implemented in Islandora 8. View it live at https://github.com/mjordan/documentation/blob/iiif_docs/docs/user-documentation/iiif.md.

# What's new?

Adds a single new page (`docs/user-documentation/iiif.md`) and one image (`docs/assets/osd_collection_mode.png`).

# How should this be tested?

Same as other documentation: read, suggest, correct, etc.

Question: how much of this should be documented as "Paged content"? @seth-shaw-unlv's https://github.com/Islandora/documentation/pull/1333 is currently under review so we should make sure there is no overlap, or that the two pages link to each other where appropriate.

# Additional Notes:

Currently, what is described in this page does not work. See https://github.com/Islandora/documentation/issues/1294 for more info.

# Interested parties
@cderven  @Islandora/8-x-committers

### CODE REVIEW

**Comment by seth-shaw-unlv** [comment]
I haven't been able to reproduce @Natkeeran's issue that you referenced in the PR description and it has been working fine for me. I did it recently to make screenshots for the Doc Sprint. So, yeah, it _is_ generally working. (I suspect it was a bad derivative.)

As to the overlap, a very brief scan of this PR shows only a tiny bit of repetition, which is fine. Although, ~~I should probably update mine to reference the IIIF documentation~~. _Update: added said reference._

File: docs/user-documentation/iiif.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  1. Adjust the options to what you want.
+  1. Click the "Save Configuration" button.
+  1. After the form is saved, navigate back down to the "Collection Mode" options and _uncheck_ the "Enable Collection Mode" checkbox.
+  1. Click the "Save Configuration" button.
+  
+  ## Looking under the hood (and beyond)
+  
+  If you want to see the raw output of the IIIF API implementations in Islandora 8, visit a node that is displaying the OpenSeadragon viewer (doesn't matter if its a single image or a paged content node like a book), and tack "manifest" onto the end of the URL, like `http://myrepo.org/node/23/manifest` and hit enter. You will see the raw JSON that IIIF-compliant viewers use to render the content.
+  
+  The really neat thing is, IIIF-compliant viewers don't need to be embedded in Islandora websites. If a viewer on another website knows the URL of a IIIF manifest like the ones that Islandora can produce, that viewer can display the content described in the manifest. Some implementations of IIIF viewers tha show off the potential to combine content from multiple IIIF servers include:
```
Comment (manez) [inline docs/user-documentation/iiif.md]: "implementations of IIIF viewers tha show off"  _that_

File: docs/user-documentation/iiif.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# IIIF (International Image Interoperability Framework)
+
+[IIIF](https://iiif.io/) is a set of specifications that provides interoperability of image-based collections across platforms. What this means for a repository platform like Islandora at a general level is that image-based objects such as (still) images and paged content can be managed by Islandora but viewed in external applications, and that Islandora can bring in image-based content from elsewhere to supplement locally managed content. If this intrigues you, see the section "Looking under the hood (and beyond)" below.
```
Comment (manez) [inline docs/user-documentation/iiif.md:3]: No suggestions, I just like the "if this intrigues you" hook 😄

File: docs/user-documentation/iiif.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Zoom, pan, and rotate images within OpenSeadragon
+   - Islandora uses an IIIF-compliant image server (by default, [Cantaloupe](https://cantaloupe-project.github.io/)) that utilizes the [IIIF Image API](https://iiif.io/api/image/2.1/). This capablity is similar to what Islandora 7.x users experience when they view a Large Image.
+- Display thumbnails for all pages of a book or newspaper issue within image viewers
+   - IIIF-compliant image viewers such as OpenSeadragon or Mirador can display a "collection" of images such as all the pages of a book or newspaper issue using the [IIIF Presentation API](https://iiif.io/api/presentation/2.1/). For example, here is a screenshot of OpenSeadragon rendering all the pages of a book:
+   
+   ![OpenSeadragon rendering book pages](../assets/osd_collection_mode.png)
+
+## Using IIIF in Islandora 8
+
+Implementation of the IIIF Presentation API is new in Islandora 8, and using it is as simple as configuring a Context (in fact, you don't even need to configure it, the Context, Openseadragon Block, already exists by default).
```
Comment (manez) [inline docs/user-documentation/iiif.md]: "need to configure it, the Context, Openseadragon Block, already exists by default" can this be "need to configure it; the Context, Openseadragon Block, already exists by default"? There aren't enough semi-colons in the world.

**Review by manez — CHANGES_REQUESTED** [review]
Only the typo is a true requested change.

File: docs/user-documentation/iiif.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
```
Comment (cderven) [inline docs/user-documentation/iiif.md:17]: Typo - "a paged content Islndora object" -> Islandora

File: docs/user-documentation/iiif.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+To use this Context, all that is required is for your book or newspaper (or other paged content) to be given a model of "Paged Content" or "Publication Issue". Then, in the Openseadragon Block Context, make sure the term used in the "Node has term" condition (you can register more than one term there). Now, when you view a paged content Islndora object, you will see thumbnails of all of its child pages (assuming you have added some child pages to the object) in the OpenSeadragon viewer as illustrated above. 
+
+You can change how the paged content thumbnails are arranged in the OpenSeadragon viewport by doing the following:
```
Comment (cderven) [inline docs/user-documentation/iiif.md:20]: Hi Mark - running through the steps below, I can't seem to successfully change "collection layout" from vertical to horizontal. Going through the steps, the thumbnails are still rendering as vertical in the viewer.

File: docs/user-documentation/iiif.md (hunk 3 above)
Comment (cderven) [reply docs/user-documentation/iiif.md]: I agree, the semi-colon makes the sentence read better...

**Comment by cderven** [comment]
I don't there's any direct overlap here with paged content and I don't think anything needs to be moved to paged content.

File: docs/user-documentation/iiif.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+   - IIIF-compliant image viewers such as OpenSeadragon or Mirador can display a "collection" of images such as all the pages of a book or newspaper issue using the [IIIF Presentation API](https://iiif.io/api/presentation/2.1/). For example, here is a screenshot of OpenSeadragon rendering all the pages of a book:
+   
+   ![OpenSeadragon rendering book pages](../assets/osd_collection_mode.png)
+
+## Using IIIF in Islandora 8
+
+Implementation of the IIIF Presentation API is new in Islandora 8, and using it is as simple as configuring a Context (in fact, you don't even need to configure it, the Context, Openseadragon Block, already exists by default).
+
+To use this Context, all that is required is for your book or newspaper (or other paged content) to be given a model of "Paged Content" or "Publication Issue". Then, in the Openseadragon Block Context, make sure the term used in the "Node has term" condition (you can register more than one term there). Now, when you view a paged content Islndora object, you will see thumbnails of all of its child pages (assuming you have added some child pages to the object) in the OpenSeadragon viewer as illustrated above.
```
Comment (cderven) [inline docs/user-documentation/iiif.md]: For clarity, could we have a link at the end of the first sentence to Paged Content?

File: docs/user-documentation/iiif.md (hunk 1 above)
Comment (mjordan) [reply docs/user-documentation/iiif.md]: Done

File: docs/user-documentation/iiif.md (hunk 3 above)
Comment (mjordan) [reply docs/user-documentation/iiif.md]: Done.

File: docs/user-documentation/iiif.md (hunk 4 above)
Comment (mjordan) [reply docs/user-documentation/iiif.md:17]: Done.

File: docs/user-documentation/iiif.md (hunk 6 above)
Comment (mjordan) [reply docs/user-documentation/iiif.md]: Done.

File: docs/user-documentation/iiif.md (hunk 5 above)
Comment (mjordan) [reply docs/user-documentation/iiif.md:20]: I'm afraid I'm having some problems with OpenSeadragon at the moment. I wonder if someone else like @manez , @seth-shaw-unlv , or @dannylamb  can try to replicate what you're seeing.

Comment (seth-shaw-unlv) [reply docs/user-documentation/iiif.md:20]: I am not able to reproduce @cderven's issue. Changing the selection, saving the settings, and refreshing the _Paged Content_ node's page is allowing me to toggle back and forth with no issues.

Perhaps you need to clear your cache or do a hard refresh on your browser?

Comment (seth-shaw-unlv) [reply docs/user-documentation/iiif.md:20]: Are there any error messages appearing in your Drupal error logs (`/admin/reports/dblog`) or in the browser's JS console?

Comment (cderven) [reply docs/user-documentation/iiif.md:20]: Thanks @seth-shaw-unlv, this was a browser issue and I can verify @mjordan's description now of adjusting these settings. I'll mark this as resolved.

**Comment by mjordan** [comment]
@manez I've lost track of where the remaining typo is - can you clarify?

**Comment by manez** [comment]
@mjordan you got it already! It's all good now 😄 

Edit: I thought I had just left a comment, not requested changes. Cleared that now too.

**Review by manez — APPROVED** [review]

**Pull request merged**